Frameworks/AddressBook/ABMultiValue.mm

```diff
@@ -19,74 +19,108 @@
 #import <StubReturn.h>
 #import "AssertARCEnabled.h"
 
+#import "ABMultiValueInternal.h"
+
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 CFTypeRef ABMultiValueCopyValueAtIndex(ABMultiValueRef multiValue, CFIndex index) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    if (multiValue == nullptr) {
+        return nullptr;
+    }
+
+    _ABMultiValue* multi = (__bridge _ABMultiValue*)multiValue;
+    return [multi copyValueAtIndex:index];
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 CFArrayRef ABMultiValueCopyArrayOfAllValues(ABMultiValueRef multiValue) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    if (multiValue == nullptr) {
+        return nullptr;
+    }
+
+    _ABMultiValue* multi = (__bridge _ABMultiValue*)multiValue;
+    return [multi copyArrayOfAllValues];
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 CFIndex ABMultiValueGetCount(ABMultiValueRef multiValue) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    if (multiValue == nullptr) {
+        return 0;
+    }
+
+    _ABMultiValue* multi = (__bridge _ABMultiValue*)multiValue;
+    return [multi getCount];
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 CFIndex ABMultiValueGetFirstIndexOfValue(ABMultiValueRef multiValue, CFTypeRef value) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    if (multiValue == nullptr) {
+        return -1;
+    }
+
+    _ABMultiValue* multi = (__bridge _ABMultiValue*)multiValue;
+    return [multi getFirstIndexOfValue:value];
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 CFStringRef ABMultiValueCopyLabelAtIndex(ABMultiValueRef multiValue, CFIndex index) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    if (multiValue == nullptr) {
+        return nullptr;
+    }
+
+    _ABMultiValue* multi = (__bridge _ABMultiValue*)multiValue;
+    return [multi copyLabelAtIndex:index];
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 ABMultiValueIdentifier ABMultiValueGetIdentifierAtIndex(ABMultiValueRef multiValue, CFIndex index) {
```

Comment:
No, just within a single multivalue -- it's mainly to differentiate pairs within a multivalue that might share a label.

---
In reply to: [75181025](https://github.com/Microsoft/WinObjC/pull/884#discussion_r75181025) [](ancestors = 75181025)